appinfo/application.php

```diff
@@ -84,8 +84,10 @@ public function __construct($params=[]) {
 			$config = $c->getServer()->getConfig();
 			$urlGenerator = $c->getServer()->getURLGenerator();
 			$appManager = $c->getServer()->getAppManager();
+			$l10nFactory = $c->getServer()->getL10NFactory();
```

Comment:
I suggest this whole commit be made in a separate PR as it is unrelated to the _spreed-integration_, i.e. is also required for the Contacts app without Spreed.

Reply:
Optimally this change should land in the nextcloud/server directly and provide an API endpoint to retrieve all the languages. This code is basically a copy from the nextcloud settings app(?), we should be as DRY as possible.